src/openrct2/world/MapAnimation.cpp

```diff
@@ -457,7 +457,7 @@ static bool map_animation_invalidate_large_scenery(const CoordsXYZ& loc)
             continue;
 
         auto* sceneryEntry = tileElement->AsLargeScenery()->GetEntry();
-        if (sceneryEntry->flags & LARGE_SCENERY_FLAG_ANIMATED)
+        if (sceneryEntry && sceneryEntry->flags & LARGE_SCENERY_FLAG_ANIMATED)
         {
```

Comment:
Can you make it explicit sceneryEntry != nullptr

Reply:
Done, not sure if I need a new mergerequest now

Reply:
Nope, you pushed to the same branch, so it appears right here automatically :)